lang/cs/build.props

```diff
@@ -141,11 +141,11 @@ under the License.
       DestinationFiles="@(MySourceFiles->'$(TargetDir)%(Filename)%(Extension)')"
     />
   </Target>
-    <ItemGroup>
+    <!--<ItemGroup>
```

Comment:
Why is this changed? Also, please either remove the code or don't, don't leave code commented out.